### PR TITLE
make assessment only page more user centred

### PR DESCRIPTION
**Pull request by EmmaFrith**
before:

Screenshot 2021-05-25 at 16 14 06

after:

Screenshot 2021-05-25 at 16 14 24

### CODE REVIEW

File: app/views/content/assessment-only-providers.md [hunk 1]
```diff
@@ -466,7 +466,8 @@ keywords:
   - experienced
 ---
 
-A number of accredited providers have been approved to offer the 
-Assessment Only (AO) route to qualified teacher status (QTS). 
-If you wish to apply for the AO route you should apply directly to one if the 
-following approved providers:
+If you’re an experienced teacher with a degree, you may be able to get qualified teacher status (QTS) through a 12-week 'assessment only' (AO) programme.
```
Comment (peteryates) [inline app/views/content/assessment-only-providers.md:469]:
```suggestion
If you’re an experienced teacher with a degree, you may be able to get qualified teacher status (QTS) through a 12-week assessment only (AO) programme.
```

I think as it's already been quoted in the heading it looks a bit funny here too.

**Review by peteryates — DISMISSED** [review]
AO, let's go!

**Review by peteryates — APPROVED** [review]
🚚

**Review by MylesJarvis — APPROVED** [review]
Let's go

**Pull request merged**